julia/oxygen/server.jl

```diff
@@ -0,0 +1,20 @@
+using Pkg
+Pkg.activate(pwd())
+
+using Oxygen
+using HTTP
+
+@get "/" function()
+    ""
+end
+
+@get "/user/{id}" function(request::HTTP.Request, id)
+    id
+end
+
+@post "/user" function ()
+   ""
+end
+
+# Could be faster setting access_log = nothing and using multithreading with serveparallel
```

Comment:
Maybe you could implement this, and also make it for merly.

I don't know about Julia usage, but multi threading seems to be, in some languages, done in specialized libraries, like gunicorn in python, or in explicit implementation like cluster.ts for bun in javascript

Reply:
[78dde22](https://github.com/the-benchmarker/web-frameworks/pull/7286/commits/78dde2266f45d603d35dc862d3f6fd95cd0ab974)